### PR TITLE
Adds identity to desert riders, makes grenzels consistant

### PR DESCRIPTION
**Pull request by notafurryy**
## About The Pull Request

Adds some lore and regional identity to desert riders rather than being vaguely arabic assassins and forces half-kin to also be grenzels.

Lore by albo3 on ratwood discord

## Why It's Good For The Game

Expands the lore and adds identity to factions while not being gameplay intrusive

### CODE REVIEW

**Comment by notafurryy** [comment]
whoops forgot that the definitions added in my spellblade update are not in main yet

File: code/modules/jobs/job_types/roguetown/mercenaries/types/grenzelhoft.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	name = "Grenzelhoft Mercenary"
 	tutorial = "Experts, Professionals, Expensive. Those are the first words that come to mind when the emperiate Grenzelhoft mercenary guild is mentioned. While you may work for coin like any common sellsword, mantaining the prestige of the guild will be of utmost priority."
 	allowed_sexes = list(MALE, FEMALE)
-	allowed_races = list(
+	no = list(
```
[the remark this answers is not in the document]
Comment (EvenInDeathIStillServe) [reply code/modules/jobs/job_types/roguetown/mercenaries/types/grenzelhoft.dm:5]: The code ain't coding.

[the remark this answers is not in the document]
Comment (notafurryy) [reply code/modules/jobs/job_types/roguetown/mercenaries/types/grenzelhoft.dm]: I have zero clue how that happened

**Review by Gyrandola — APPROVED** [review]

**Review by Gyrandola — APPROVED** [review]

**Comment by albo32** [comment]
there's actually an error I made when writing it--I wrote "Zybantine" instead of "Zybantian", which is incorrect

**Pull request merged**